docs/bundles/SyliusFixturesBundle/builtin_listeners.rst

```diff
@@ -0,0 +1,80 @@
+Built-in listeners
+==================
+
+**SyliusFixturesBundle** comes bundled with a few useful listeners.
```

Comment:
A Bundle comes bundled? Nope. :)
`The **SyliusFixturesBundle** comes with a few useful listeners.`